src/lib/cache/subCache.cpp

```diff
@@ -30,12 +30,19 @@
 #include "common/sem.h"
 #include "common/string.h"
 #include "apiTypesV2/HttpInfo.h"
+#include "apiTypesV2/ngsiWrappers.h"
 #include "mongoBackend/MongoGlobal.h"
 #include "mongoBackend/mongoSubCache.h"
 #include "ngsi10/SubscribeContextRequest.h"
 #include "cache/subCache.h"
 #include "alarmMgr/alarmMgr.h"
 
+#include "apiTypesV2/Subscription.h" // FIXME PR: remove after moving things to ngsiWrapper
```

Comment:
Old this stuff should be needed after "fixing" ngsiWrapper.cpp/h. I'll remove this.